pages/about/index.js

```diff
@@ -139,6 +139,12 @@ const TEAM_MEMBERS = [
     pronouns: "She/Her",
     src: "assets/headshots/christina_tremel.png",
   },
+  {
+    name: "Susanne Gov",
+    title: "Application Analyst",
+    pronouns: "she/her",
```

Comment:
maybe change to "She/Her" to follow the pattern of the others

Reply:
thank you, i copied/pasted and didnt pay attention to the capitalization. fixed!